src/libecalc/presentation/yaml/validation_errors.py

```diff
@@ -179,6 +179,18 @@ def as_pydantic_compatible(self) -> PydanticLoc:
         return tuple(keys)
 
 
+@dataclass
+class FileMark:
+    line: int
```

Comment:
```suggestion
    line_number: int
    column_number: int
```

? Or is that given? datatypes in python are just type hints...

Reply:
It's obvious to me, now, since I just implemented this...